### ISSUE TITLE
missing "/local/" in path on init?

### ISSUE THREAD

**Issue by Elkenfugel** [ovenwerks/studio-controls#45]
i ran in bash
`studio-controls`
and this came back:
```
Traceback (most recent call last):
  File "/usr/local/bin/studio-controls", line 1420, in <module>
    us = StudioControls()
  File "/usr/local/bin/studio-controls", line 290, in __init__
    builder.add_from_file("/usr/share/studio-controls/studio-controls.glade")
gi.repository.GLib.Error: g-file-error-quark: Failed to open file “/usr/share/studio-controls/studio-controls.glade”: No such file or directory (4)
```
I believe that line 290 should read : 

builder.add_from_file("/usr/**local**/share/studio-controls/studio-controls.glade")

**Comment by ovenwerks** [comment]
good catch /usr sh9ould be replaced with {install_path}

**Comment by ovenwerks** [comment]
This should be fixed in 2.1.1
